libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/serial_api.c

```diff
@@ -207,6 +245,11 @@ void UART1_Handler()
 {
     uart_irq(UART_1, 1);
 }
+
+void UART2_Handler()
```

Comment:
I'd like to use NVIC Set vector API but I can't. Because W7500 chip does not support function which W7500 remap it's vector table. So I refer to HAL of NORDIC(NRF51822) and use weak definitions.
If do you have another method for this issue, Please let me know.